src/skmultiflow/trees/hoeffding_adaptive_tree_regressor.py

```diff
@@ -177,16 +179,29 @@ def _partial_fit(self, X, y, weight):
             self._active_leaf_node_cnt = 1
         self._tree_root.learn_from_instance(X, y, weight, self, None, -1)
 
-    def get_normalized_error(self, prediction, y):
-        normal_prediction = self.normalize_target_value(prediction)
-        normal_value = self.normalize_target_value(y)
-        return np.abs(normal_value-normal_prediction)
-
     def filter_instance_to_leaves(self, X, y, weight, split_parent, parent_branch, update_splitter_counts):
         nodes = []
         self._tree_root.filter_instance_to_leaves(X, y, weight, split_parent, parent_branch,
                                                   update_splitter_counts, nodes)
         return nodes
 
+    # Override HoeffdingTreeClassifier
```

Comment:
The comment is unnecessary IMO.

Reply:
The comment was removed in commit [f5838ee](https://github.com/scikit-multiflow/scikit-multiflow/pull/220/commits/f5838ee9b96b27c674a13a529e065d3b91c38aa2).